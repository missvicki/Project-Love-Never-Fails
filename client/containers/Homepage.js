/* eslint-disable react/prefer-stateless-function */
import React, { Component } from "react";
import { connect } from "react-redux";

import Header from "../components/common/Header";
import Footer from "../components/common/Footer";

class Homepage extends Component {
  render() {
    return (
      <div>
        <Header />
        <Footer />
      </div>
    );
  }
}

export default Homepage;
