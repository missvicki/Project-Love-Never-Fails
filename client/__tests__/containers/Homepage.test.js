import React from "react";
import { shallow } from "enzyme";
import Homepage from "../../containers/Homepage";

describe("<Homepage />", () => {
  const wrapper = shallow(<Homepage />);
  it("should render Homepage without crashing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
