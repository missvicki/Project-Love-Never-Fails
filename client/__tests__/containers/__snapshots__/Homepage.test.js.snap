// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Homepage /> should render Homepage without crashing 1`] = `
<div>
  <Header />
  <Footer />
</div>
`;
